docs/src/pages/components/dialogs/dialogs.md

```diff
@@ -121,6 +121,10 @@ Try the demo below to see what we mean:
 
 {{"demo": "pages/components/dialogs/ScrollDialog.js"}}
 
+## Dialog with portals inside
+
+{{"demo": "pages/components/dialogs/DialogWithPortalsInside.js"}}
```

Comment:
Having a new demo is interesting but I don't think that it should be on the Dialog page. It could either be on the Modal page or a new one for the TrapFocus component.